pinakes/main/approval/models.py

```diff
@@ -10,6 +10,36 @@
 models.CharField.register_lookup(Length)
 
 
+class Notification(models.Model):
+    """Notification model"""
+
+    name = models.CharField(
+        max_length=128,
+        editable=True,
+        help_text="Name of the notification method",
+    )
+    setting_schema = models.JSONField(
```

Comment:
@bzwei Is this JSONField storing the schema and the data (default values) or is the data stored separately. Since the data can contain passwords they would have to be stored encrypted.

Reply:
Data is stored separately in template. Yes encryption is kept in mind although not implemented yet.